components/automate-gateway/api/nodes/nodes.proto

```diff
@@ -69,6 +96,12 @@ service NodesService {
 			action: "infra:nodes:delete"
 		};
 	};
+
+	/*
+	Bulk delete by id
+
+	Deletes a set of nodes given a list of ids.
```

Comment:
```suggestion
	Deletes a set of nodes that match a list of ids.
```
What happens if one of the listed ids does not exist?

Reply:
> What happens if one of the listed ids does not exist?

i think we just ignore and move on but ill check